content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -104,6 +127,11 @@ for p in $(kubectl get pods -l jobgroup=jobexample -o name)
 do
   kubectl logs $p
 done
+```
+
+The output is:
```

Comment:
nit: for consistency reason, would you agree is good to keep `similar to this:` in ?